cdm/src/main/java/ucar/nc2/ft2/coverage/adapter/SimpleGeometryCSBuilder.java

```diff
@@ -56,14 +57,13 @@ public static String describe(NetcdfDataset ds, CoordinateSystem cs, Formatter e
     return fac.type == null ? "" : fac.showSummary();
   }
 
-  ////////////////////////////////////////////////////////////////////////////////////
+  
```

Comment:
Are you referring to the variables with package visibility in that class?

Obviously getters and setters are advantageous in Java and C# - that's very fundamental to those two languages- you don't need articles describing something like that and it's honestly a little insulting to suggesting that I go "read some articles" on something you probably learn in CS 200.

So why are they not getters and setters? THREDDS Convention. That's it. Maybe after some further refactor it will be reworked (and in fact I thought about writing those) but seems mkind of out of scope if this ticket.

Reply:
Probably will go add those setters and getters, thanks for the reminder Carl!

Reply:
Thanks for chatting about this in person.

Reply:
Thanks for providing great feedback! The changes to get getters set up actually didn't inflict that much problems, and setters are actually not that useful here so they were omitted (the coord sys is supposed to be pretty immutable at this point it seems).